Lib/multiprocessing/connection.py

```diff
@@ -1011,8 +1011,18 @@ def _exhaustive_wait(handles, timeout):
         # returning the first signalled might create starvation issues.)
         L = list(handles)
         ready = []
+        if len(L) > 60:
```

Comment:
In my personal view, it's better to add a note to explain why we need to use `BatchedWaitForMultipleObjects` if the handles length is larger than 60. Or it could become a magic number in the codes.

Reply:
```suggestion
        # Windows limits `WaitForMultipleObjects` at 64 handles, and we use a
        # few for synchronisation, so we switch to batched waits at 60.
        if len(L) > 60:
```